docs: propagate Python to all parts of the Platform documentation (#711)
apify/apify-docs: sources/platform/integrations/webhooks/actions.md

| `eventData` | Object | Data associated with the trigger event, [see Events](./events.md). |
| `resource`  | Object | The resource that caused the trigger event, [see below](#resource).                 |

#### Resource

> The `resource` variable represents the triggering system resource. For example when using the `ACTOR.RUN.SUCCEEDED` event, the resource is the Actor run. The variable will be replaced by an `Object` that one would receive as response from the relevant API at the moment when the webhook is triggered. So for the Actor run resource, it would be the response of the [Get Actor run](/api/v2#/reference/actors/run-object-deprecated/get-run) API endpoint.

Reviewer: ```suggestion
The `resource` variable represents the triggering system resource. For example, when using the `ACTOR.RUN.SUCCEEDED` event, the resource is the Actor run. The variable will be replaced by the `Object` that you would receive as a response from the relevant API at the moment when the webhook is triggered. So for the Actor run resource, it would be the response of the [Get Actor run](/api/v2#/reference/actors/run-object-deprecated/get-run) API endpoint.
```
